pkg/utils/nodeclaim/nodeclaim.go

```diff
@@ -232,3 +201,10 @@ func UpdateNodeOwnerReferences(nodeClaim *v1beta1.NodeClaim, node *v1.Node) *v1.
 	})
 	return node
 }
+
+func GetCondition(nc *v1beta1.NodeClaim, match apis.ConditionType) apis.Condition {
```

Comment:
This actually already exists with `nc.StatusConditions().Get()` so you don't need to create a separate function for it. The only reason that we have one of these here for nodes is because it uses a different api object/interface for status conditions